src/layout/Cell.js

```diff
@@ -139,6 +148,38 @@
             })
         ;
         dragHandles.exit().remove();
+        
+        
+    };
+
+    Cell.prototype.onMouseEnter = function (widgetArr){
+        var arr = widgetArr;
+        var opacity = this.indicatorOpacity();
+        var bgColor = this.indicatorColor();
+        return function(){
+            for(var i in arr){
+                var node = document.getElementById(arr[i]);
+                var rect = node.getBoundingClientRect();
+                var div = document.createElement("div");
+                this.appendChild(div);
+                div.className = "update-indicator";
+                div.style.position = "fixed";
+                div.style.top = rect.top + "px";
+                div.style.left = rect.left + "px";
+                div.style.width = rect.width + "px";
+                div.style.height = rect.height + "px";
+                div.style.opacity = opacity;
+                div.style.padding = "0px";
+                div.style.zIndex = 1000;
+                div.style.backgroundColor = bgColor;
```

Comment:
I would rather you used ```d3.append().style({xxx:yyy});``` paradigm:
* d3 fixes a few cross browser inconsistencies 
* We could then use transitions if we wanted to
* general code consistency